docs/definition.rst

```diff
@@ -204,7 +210,7 @@ The following keys are valid for this section:
             package_pip: https://github.com/example_user/ansible/archive/refs/heads/ansible.tar.gz
 
     ``ansible_runner``
-      The version of the Ansible Runner Python package to be installed. This value
+      The version of the Ansible Runner Python package is to be installed. This value
```

Comment:
I think that if the `is` is added, another `that` needs to be added before it. I think it's better to leave both `is` and `that` away though. I'm not 100% sure though, English isn't my native language :)

(Same for the next change below.)